charts/rancher-thanos/v0.0.4/templates/exporter-deployment.yaml

```diff
@@ -0,0 +1,70 @@
+{{- if .Values.exporter.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: {{ template "app.fullname" . }}
+    chart: {{ .Chart.Name }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    component: exporter
+  name: exporter-{{ .Release.Name }}
+spec:
+  replicas: {{ .Values.exporter.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "app.fullname" . }}
+      chart: {{ .Chart.Name }}
+      release: {{ .Release.Name }}
+      component: exporter
+  template:
+    metadata:
+      labels:
+        app: {{ template "app.fullname" . }}
+        chart: {{ .Chart.Name }}
+        release: {{ .Release.Name }}
+        component: exporter
+    spec:
+      containers:
+      - name: exporter
+        image: {{ template "system_default_registry" . }}{{ .Values.exporter.image.repository }}:{{ .Values.exporter.image.tag }}
+        imagePullPolicy: {{ .Values.exporter.image.pullPolicy }}
+        env:
+        - name: CATTLE_API_TOKEN
+          value: {{ .Values.apiToken }}
+        - name: CATTLE_URL
+          value: https://{{ .Values.rancherHost }}
```

Comment:
prefer to use `https://{{ trimPrefix "https://" .Values.rancherHost }}` for better fault input tolerance.

Reply:
Updated